test/unit/driver-specs.js

```diff
@@ -21,7 +21,7 @@ describe('driver.js', () => {
   describe('createSession', () => {
     it('should throw an error if app can not be found', async () => {
       let driver = new AndroidUiautomator2Driver({}, false);
```

Comment:
yes, better to stub out `driver.fillDeviceDetails` here rather than add a cap used just for testing